src/components/pages/FamilySettings/RenderFamilySettings.js

```diff
@@ -29,12 +30,15 @@ const FamilySettings = props => {
   };
 
   return (
-    <Layout className="parent-dashboard add-child">
+    <Layout className="edit-players">
```

Comment:
Looking at what the component does this seems to make more sense than the previous class name.